### PR TITLE
Updated instructions for jenkins-slave-npm

### PR DESCRIPTION
**Pull request by oybed**

### CODE REVIEW

**Comment by gsampaio-rh** [comment]
LGTM

File: exercises/1-the-manual-menace/README.md [hunk 1]
```diff
@@ -436,7 +436,21 @@ where
     * `<GIT_URL>` is the full clone path of the repo where this project is stored (including the https && .git)
     * `<YOUR_NAME>` is the prefix for your `-ci-cd` project.
 
-8. Create a new object `ci-cd-builds` in the Ansible `inventory/host_vars/ci-cd-tooling.yml` to drive the s2i build configuration.
+8. Create `params/jenkins-s2i-slaves` and add the following content; replacing variables as appropriate.
+
```
Comment (springdo) [inline exercises/1-the-manual-menace/README.md]: I think this should not be called `s2i-slaves` as it's not s2i, it's a jenkins slave. The param file should just be called `jenkins-slave-npm`

File: exercises/1-the-manual-menace/README.md [hunk 2]
```diff
@@ -436,7 +436,21 @@ where
...
+<kbd>📝 *enablement-ci-cd/params/jenkins-s2i-slaves*</kbd>
+```
+IMAGE_STREAM_NAME=jenkins-slave-npm
+IMAGE_STREAM_NAMESPACE=<YOUR_NAME>-ci-cd
+IMAGE_STREAM_ROLE=jenkins-slave
```
Comment (springdo) [inline exercises/1-the-manual-menace/README.md]: If we use Params from vars, we can not have people edit YOUR_NAME and also get reuse of the string `jenkins-slave-npm` by setting it to a variable

File: exercises/1-the-manual-menace/README.md [hunk 3]
```diff
@@ -466,19 +486,19 @@ git commit -m "Adding Jenkins and Jenkins s2i"
 ```bash
 git push
 ```
-10.  In order for Jenkins to be able to run `npm` builds and installs we must configure a `jenkins-build-slave` for Jenkins to use. This slave will be dynamically provisioned when we run a build. It needs to have NodeJS and npm installed in it. These slaves can take a time to build themselves so to speed up we have placed the slave within OpenShift and an ImageStream, with the "role=jenkins-slave" label, is added by the Configuration-as-Code run managed by the openshift-applier run below.
+11.  In order for Jenkins to be able to run `npm` builds and installs we must configure a `jenkins-build-slave` for Jenkins to use. This slave will be dynamically provisioned when we run a build. It needs to have NodeJS and npm installed in it. These slaves can take a time to build themselves so to speed up we have placed the slave within OpenShift and an ImageStream, with the "role=jenkins-slave" label, is added by the Configuration-as-Code run managed by the openshift-applier run below.
```
Comment (springdo) [inline exercises/1-the-manual-menace/README.md]: Move this stuff up to where we introduce the node js slave

File: exercises/1-the-manual-menace/README.md (hunk 2 above)
Comment (oybed) [reply exercises/1-the-manual-menace/README.md]: agreed

**Comment by oybed** [comment]
@springdo @gsampaio-rh created PR to add param file to support the latest iteration of this PR: https://github.com/rht-labs/enablement-ci-cd/pull/45

**Pull request merged**